lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -116,7 +134,8 @@ private static IConfiguration GetRuntimeConfiguration(string name)
                         .Set(AzureBlockBlobFileSystemConfiguration.ConnectionString, connectionString)
                         .Build();
                 case AzureBatch:
-                    return AzureBatchRuntimeClientConfiguration.ConfigurationModule
+                    List<string> ports = new List<string> { "2000", "2001", "2002", "2003", "2004" };
+                    return AzureBatchRuntimeClientConfiguration.GetConfigurationModule(ports)
```

Comment:
>ports [](start = 87, length = 5)

Why are we binding ports here since they are already bound on line 150?

Reply:
Removed duplicate binding. Used AzureBatchPoolDriverPortsList binding directly where needed.

---
In reply to: [201155017](https://github.com/sharathmalladi/reef/pull/5#discussion_r201155017) [](ancestors = 201155017)